include/oneapi/dpl/pstl/tuple_impl.h

```diff
@@ -793,6 +793,25 @@ struct __decay_with_tuple_specialization<::std::tuple<_Args...>>
 template <typename... _Args>
 using __decay_with_tuple_specialization_t = typename __decay_with_tuple_specialization<_Args...>::type;
 
+// Flatten nested std::tuple or oneapi::dpl::__internal::tuple types into a single std::tuple.
+template <typename _T>
+struct __flatten_std_or_internal_tuple
```

Comment:
Optional suggestion: `__flatten_std_or_internal_tuple` -> `__flatten_tuple`.

Reply:
This utility has been removed